e2e-test/cypress/integration/currency_format_spec.js

```diff
@@ -0,0 +1,47 @@
+
+describe("Checking format for Value and currency Symbol of all languages", function () {
+  let languageArray = [
+    {
```

Comment:
what if I want to use the language or format in other tests? I don't want to import something from another test